hedera-node/hedera-mono-service/src/main/java/com/hedera/services/store/contracts/precompile/proxy/ViewExecutor.java

```diff
@@ -204,7 +217,7 @@ private Bytes answerGiven(final int selector) {
             case ABI_ID_GET_TOKEN_KEY -> {
                 final var wrapper = GetTokenKeyPrecompile.decodeGetTokenKey(input);
                 validateTrueOrRevert(
-                        stateView.tokenExists(wrapper.tokenID()),
+                        ledgers.tokens().contains(wrapper.tokenID()),
```

Comment:
I think we can create a new method `exists` inside the WorldLedgers. In the case of static frame, it would check whether the token is present, using a method from StaticEntityAccess and if it's a mutable operation, it would check the existence using tokensLedger. Then we can use this new method to check the token for existence in all ViewExecutor cases where it's needed.